base/stat.jl

```diff
@@ -53,8 +53,8 @@ StatStruct(buf::Union{Vector{UInt8},Ptr{UInt8}}) = StatStruct(
     ccall(:jl_stat_size,    UInt64,  (Ptr{UInt8},), buf),
     ccall(:jl_stat_blksize, UInt64,  (Ptr{UInt8},), buf),
     ccall(:jl_stat_blocks,  UInt64,  (Ptr{UInt8},), buf),
-    ccall(:jl_stat_mtime,   Float64, (Ptr{UInt8},), buf),
-    ccall(:jl_stat_ctime,   Float64, (Ptr{UInt8},), buf),
+    ccall(:jl_stat_mtime,   Float64, (Ptr{UInt8},), buf) |> Dates.unix2datetime,
```

Comment:
I've changed the struct. I'm not sure if DateTimes even exists yet when the c code is run?